lib/balance-tx/lib/Cardano/Wallet/Write/Tx/SizeEstimation.hs

```diff
@@ -697,3 +701,14 @@ mkLedgerTxOut txOutEra address bundle =
         RecentEraConway -> W.toConwayTxOut txOut
       where
         txOut = W.TxOut address bundle
+
+data TxWitnessTag
+    = TxWitnessByronUTxO
+    | TxWitnessShelleyUTxO
+    deriving (Show, Eq)
+
+assumedTxWitnessTag :: UTxOAssumptions -> TxWitnessTag
+assumedTxWitnessTag = \case
```

Comment:
Why not leave it in the UtxoAssumptions module?

Reply:
🤔 `TxWitnessTag` is ultimately only needed for `SizeEstimation.hs`-functions, and seems more internal to me. I'd favor not putting it in `Write.UTxOAssumption` now.